packages/odyssey-storybook/src/components/Infobox/Infobox.stories.tsx

```diff
@@ -52,24 +64,157 @@ const Template: Story<InfoboxProps> = ({ heading, variant }) => (
   />
 );
 
+const FormTemplate: Story<InfoboxProps> = ({
+  heading,
+  variant,
+  content,
+  actions,
+}) => (
+  <Form heading="Sign in">
+    <Infobox
+      heading={heading}
+      variant={variant}
+      content={content}
+      actions={actions}
+    />
+    <TextInput type="text" label="Username" disabled />
+    <TextInput type="password" label="Authorization code" disabled />
+    <Form.Actions>
+      <Button variant="primary" disabled>
+        Login
+      </Button>
+    </Form.Actions>
+  </Form>
+);
+
+const TableTemplate: Story<InfoboxProps> = ({
+  heading,
+  variant,
+  content,
+  actions,
+}) => (
+  <Table
+    screenReaderCaption="Information about the largest and smallest planets."
+    caption="Big and small planets"
+    withContainer={true}
+  >
+    <Table.Header>
+      <Table.Row>
+        <Table.HeaderCell scope="col">Planet</Table.HeaderCell>
+        <Table.HeaderCell scope="col" format={"num"}>
+          Radius (km)
+        </Table.HeaderCell>
+        <Table.HeaderCell scope="col">Type</Table.HeaderCell>
+        <Table.HeaderCell scope="col" format={"date"}>
+          Perihelion date
+        </Table.HeaderCell>
+      </Table.Row>
+    </Table.Header>
+    <Table.Body>
+      <Table.Row>
+        <Table.DataCell>Jupiter</Table.DataCell>
+        <Table.DataCell format={"num"}>69,911</Table.DataCell>
+        <Table.DataCell>Gas giant</Table.DataCell>
+        <Table.DataCell format={"date"}>January 21, 2023</Table.DataCell>
+      </Table.Row>
+      <Table.Row>
+        <Table.DataCell>
+          Pluto
+          <Infobox
+            heading={heading}
+            variant={variant}
+            content={content}
+            actions={actions}
+          />
+        </Table.DataCell>
+        <Table.DataCell format={"num"}>6,371</Table.DataCell>
+        <Table.DataCell>Terrestrial</Table.DataCell>
+        <Table.DataCell format={"date"}>January 2, 2021</Table.DataCell>
+      </Table.Row>
+      <Table.Row>
+        <Table.DataCell>Mercury</Table.DataCell>
+        <Table.DataCell format={"num"}>1,737</Table.DataCell>
```

Comment:
minor but worth mentioning quickly that JSX supports passing double quote string arguments directly! no need for the wrapping { } braces